### ISSUE TITLE
Error: Music Library needs to rescan tags from files. Would you like to scan now ?

### ISSUE THREAD

**Issue by Ribero** [croneter/PlexKodiConnect#561]
**Issue description:** 

**Configuration:**

- Kodi Leila Beta 5 on macOS 10.12.6
- PKC 2.4.9
- Plex PMS Version 1.13.9.5456-ecd600442

**How to reproduce:** 

- macOS: delete ~/Library/Application Support/Kodi -> to be sure to run a fresh Kodi install
- Kodi: Install PKC repo (1.0.2) from zip
- Kodi: Install PKC (2.4.9)
- PKC: Register PKC with plex server
- PKC: Select Direct Path
- PKC: Transform UNC Library Path -> Yes
- PKC: Replace custom library path -> No
- PKC: Network Credential -> No (no need as using local direct path)
- PKC: Disable Plex Music Library -> No
- PKC: Fan art in background -> Yes
- PKC: Custom ratings -> No
- PKC: Fine tune PKC -> No
- PKC: Wait for Kodi restart
- PKC: Select Plex User
- PKC: Message: New Plex Music library detected, sorry but we need to restart -> OK (but sync already started)
- Kodi: Wait sync to finish (Movies, TV Shows then music)
- Kodi: Exit Kodi Settings
- Kodi: Go to Kodi main menu (Movies: Ok, TV SHows: Ok)
- Kodi: Go to music -> Music or Music -> Albums -> Error message but title plays correctly.
- Kodi: Quit Kodi
- Kodi: Restart Kodi
- Kodi: Go to music -> Music or Music -> Albums -> Error message but title plays correctly.
- PKC: PKC Settings -> Advanced -> "Partial or Full Reset of Database and PKC"
- Kodi: Quit Kodi
- Kodi: Restart Kodi
- Kodi: Go to music -> Music or Music -> Albums -> Error message but title plays correctly.

**Attachments:**

- [PKC Settings](url
https://discourse-cdn-sjc1.com/plex/uploads/default/original/3X/4/8/48cf1b146b5845aa653c4d7d5043bca3b3046098.zip)
- Log (to pu posted soon)

**Comment by Ribero** [comment]
Your comment gave me an idea. I restored a backup of a v17 kodi db & settings. I used this backup to run a v18 on it. 

Surprise: the "migrated" v17 db and settings works like a charm with v18 & PKC. No error message. 

I then restarted from a "fresh" v18 install, done the PKC install and config. Then replaced the newly created music db by the one that was migrated from a v17... the error message has gone. 

It seems something related to the music db generates this error when the db is initialized using PKC on v18.

**Comment by Ribero** [comment]
Yes, browsing music library does not generates  error message anymore. Thanks !

Unfortunately I found a new issue with this version when PKC tries to sync Plex Playlist (Error 0). I deactivated the option in the settings and it worked again. 

I'll open a new issue for this and will provide a log asap.

**Comment by croneter** [comment]
Thanks!
